src/main/resources/templates/db_scripts/schema.json

```diff
@@ -679,6 +679,12 @@
           "fieldName": "fullCallNumberNormalized",
           "sqlExpression" : "normalize_holdings_full_call_number(jsonb)",
           "sqlExpressionQuery": "normalize_call_number_string($) || '*'"
+        },
+        {
+          "fieldName": "electronicAccess",
+          "tOps": "ADD",
+          "arraySubfield": "uri",
+          "arrayModifiers": ["uri"]
```

Comment:
This looks good. There should be a unit tests for the intended `/@` query syntax because a `/@` query will fail if the entry in schema.json is missing or wrong.